include/gatekeeper_acl.h

```diff
@@ -35,6 +35,20 @@ enum {
 	NUM_FIELDS_IPV6,
 };
 
+/*
+ * Input indices for the fields above. Fields are grouped into
+ * four bytes, except the first field which must be one byte.
+ */
+enum {
+	PROTO_INPUT_IPV6,
+	DST1_INPUT_IPV6,
+	DST2_INPUT_IPV6,
+	DST3_INPUT_IPV6,
+	DST4_INPUT_IPV6,
+	TYPE_INPUT_ICMPV6,
+	NUM_INPUTS_IPV6,
+};
```

Comment:
1. Aren't these constant only need inside file `acl.c`? If so, should we move them to there?

2. Can you add a comment explaining the difference between field and input indexes? Don't forget to add an example in the comment. I'm confused.